### DISCUSSION TITLE
Feature: Podcast browser

### DISCUSSION THREAD

**Discussion by tsunulukai** [jcorporation/myMPD#844] [Ideas]
With the radio browser and the partitions milestones achieved with brio, would there be any interest in having a proper podcast browser integrated into mympd ?

I'd love to see something like the PocketCast API directly available for consumption in mympd. (unfortunately, PocketCast API doesn't seem to be officially documented, although it could easily be grabbed from the unofficial python implementation : https://github.com/nwithan8/pycketcasts/ ; https://pycketcasts.readthedocs.io/en/latest/ )

**Comment by tsunulukai** [comment]
And... I went too fast, seems the discussion was already started in #781, so there's definitely at least a bit of interest :)

**Reply by jcorporation** [reply]
Let us discuss this in #781
